app/components/Navigation.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { connect as connectFela } from 'react-fela'
+import { pipe } from 'ramda'
+import AppBar from 'material-ui/AppBar'
+import Drawer from 'material-ui/Drawer'
+import MenuItem from 'material-ui/MenuItem'
+import Divider from 'material-ui/Divider'
+
+import styles from '../styles/Navigation'
+import { FormattedMessage } from '../../lib/Intl'
+
+class Navigation extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
```

Comment:
yeaaaahhh nice ! i wanted to ask you how to do this actually - will give it a go!